### DISCUSSION TITLE
can't access the site from LAN, externally works

### DISCUSSION THREAD

**Discussion by pmaaaaa** [globaleaks/globaleaks-whistleblowing-software#3409] [Q&A]
The problem should not be in the firewall since our provider went through the entire firewall and said everything is ok.

Is there something I am missing, I have a public IP for the local server and I can access it outside LAN with domain name. I can resolve the domain from inside LAN but when i try to access the page I get

Hmmm… can't reach this page
page took too long to respond

Any ideas?

**Comment by evilaliv3** [comment]
Hello, thank you for the question.

Actually the details provided do not enable us to identify the cause.
We suspect it could be caused by a firewall or nat rule
